app/javascript/profileDropdown/flagButton.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-alert */
+export function initFlag() {
```

Comment:
This whole file was modeled closely after `blockButton.js` so we handle the two cases consistently.